chunks/tool-nix/chunk.yaml

```diff
@@ -1,5 +1,5 @@
 variants:
-  - name: "2.11.0"
+  - name: "2"
```

Comment:
@Furisto this is simply so that it does not get hard to upgrade nix going forward. We are most likely going to see updates to the minor version here and if we can just change the version of the variant, instead of having to rename the variant and any references to it from the dazzle file, maintaining will be quite a bit easier. We do the same with Rust, where we do not have to update the chunk name every couple of weeks.